src/App.js

```diff
@@ -12,8 +12,7 @@ import ErrorBoundaryContainer from './components/Error/ErrorBoundaryContainer';
 import Oops from './components/Error/Oops';
 import NotFoundPage from './components/Error/NotFoundPage';
 import ImageDatasetListContainer from "./components/SpatialViewer/ImageDatasetListContainer";
-import SpatialViewerContainer from "./components/SpatialViewer/SpatialViewerContainer";
-import { baseURL } from '../package.json';
+import packagejson from '../package.json';
```

Comment:
With latest version of react-scripts, we can no longer deconstruct objects on import